main.go

```diff
@@ -47,9 +47,10 @@ func init() {
 }
 
 func printVersion() {
-	log.Info(fmt.Sprintf("Operator Version: %s", version.Operator))
-	log.Info(fmt.Sprintf("Go Version: %s", runtime.Version()))
-	log.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
+	log.Info(fmt.Sprintf("version: v%s", version.VersionOperator))
+	log.Info(fmt.Sprintf("operator-sdk version: v%s", version.VersionSDK))
```

Comment:
I am in favor of the `Makefile` change, but would rather keep this out of Go, see [here][1] and that commit message.  Is there a benefit to storing this information in Go?

[1]: https://github.com/openshift/cincinnati-operator/commit/c95c673082138ce8a6efb92b789791a952e4dde4#diff-f6bf0ae0d21c9408c6624174da11dafb369ce306f2cdcea704285d8acc87e19eL5